### PR TITLE
[IN-95] Feat/in95 redesign feedback

### PR DESCRIPTION
**Pull request by ghost**
#### User Story
IN-95

#### Changes made

- Pair-programmed Aleem & Rene 
- Added the Eye icon, onClick change render and icon-type
- Feedback now dynamically shows details according to if user wants to see

#### Does your new code introduce new warnings on dev console?
No

#### Test Steps
- Standard posting create process
- Can click the eye icon button to see details regarding the matches

### CODE REVIEW

**Comment by tommytwm** [comment]
image
The current implementation is missing the `Summary` header. Make sure that is in and the "`1000 total students`" is of a slightly smaller header than `Summary`

**Comment by tommytwm** [comment]
The report tags overflow to the right of the screen of my macbook. For this try to make sure that the tags are able to flex vertically in order to prevent this. This can be done by making the div container that holds all of the tags flexible, and making the chips flexible as well. If you need support on this let me know, we can find an alternative. 
image

File: internify/src/components/molecules/pages/FinalReqStudents.js [hunk 1]
```diff
@@ -2,39 +2,42 @@ import React from "react";
...
+                  <ChipNote label={msg} />
+                </li>
+              );
+            }
+          })}
+        </ul>
+      </div>
+    );
+  } else {
+    return <div></div>;
```
Comment (tommytwm) [inline internify/src/components/molecules/pages/FinalReqStudents.js]: You can just return `null`

File: internify/src/components/molecules/pages/SeekingStudents.js [hunk 2]
```diff
@@ -2,28 +2,36 @@ import React from "react";
...
+              return (
+                <li key={x}>
+                  <div></div>
+                </li>
+              );
+            } else {
+              const msg = `${data[x]} students currently ${x} a job`;
+              return (
+                <li key={x}>
+                  {" "}
```
Comment (tommytwm) [inline internify/src/components/molecules/pages/SeekingStudents.js]: Again with the empty string, is this necessary? It clogs up code and we should not be pushing this

File: internify/src/components/molecules/pages/TechMatchStudents.js [hunk 3]
```diff
@@ -2,33 +2,42 @@ import React from "react";
...
+                  <ChipNote label={msg} />
+                </li>
+              );
+            }
+          })}
+        </ul>
+      </div>
+    );
+  } else {
+    return <div></div>;
```
Comment (tommytwm) [inline internify/src/components/molecules/pages/TechMatchStudents.js]: Return `null`

**Review by tommytwm — CHANGES_REQUESTED** [review]

**Review by JudeShamsi — APPROVED** [review]

**Pull request merged**